test/unit/oidc_jwt_validate_test.rb

```diff
@@ -0,0 +1,72 @@
+require 'test_helper'
+require 'ostruct'
+require 'openssl'
+require 'jwt'
+
+class OidcJwtValidateTest < ActiveSupport::TestCase
+  context '#decoded_payload?' do
+    def setup
+      skip "SSO feature is not available for Ruby < 2.4.0" unless RUBY_VERSION >= '2.4'
+      @jwk = JWT::JWK.new(OpenSSL::PKey::RSA.new(2048))
```

Comment:
Is `setup` ran for every `test` or only for the class? The test would be a lot faster if you could reuse the key.